src/app/meta-campaign/meta-campaign.component.ts

```diff
@@ -45,11 +46,16 @@ const endPipeToken = 'timeLeftToEndPipe';
   ],
 })
 export class MetaCampaignComponent implements AfterViewChecked, OnDestroy, OnInit {
+  // Campaign ID may be passed instead
+  @Input({ required: false }) private campaignSlug: string;
+  // Passed only on the fund-filtered view of this page.
+  @Input({ required: false }) private fundSlug: string;
```

Comment:
`required: false` doesn't do anything, but I like it - maybe it's a good idea to make it our coding standard that we always pass required true or false to input decorators.

Reply:
Yep – I think for new or newly-modified inputs at least, it's helpful to show we have thought about it and explain why things are optional.